### DISCUSSION TITLE
How to deal with xyce running error

### DISCUSSION THREAD

**Discussion by Cahui1** [Xyce/Xyce#63] [General]
When I install xyce and try to run it, there is such an error. What should I do?
![捕获](https://user-images.githubusercontent.com/106376916/170669557-f427df2b-dab4-4486-ab6f-c5a9bcfdea7b.PNG)

**Comment by tvrusso** [comment]
mpirun is not finding your Xyce executable.

Either add Xyce's binary directory to your PATH variable, or specify a full path to the Xyce binary on the mpirun line.
